tools/py3_unit_tests

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# To run all tests we expect to pass with Python 3:
+#
+#     tools/py3_unit_tests
+#
+# To run just the tests in tests/test_model.py and tests/test_indexing.py:
+#
+#     tools/py3_unit_tests test_model test_indexing
+
+pushd "$(dirname $0)" >/dev/null && source common.sh && popd >/dev/null
```

Comment:
do we care about a "|| exit 1" or something special here?

Reply:
I think it's ok as-is. This is consistent with its Python 2 counterpart (`tools/unit_tests`), and, if `common.sh` doesn't get run for whatever reason, it'll fail anyway, because `common.sh` is what adds a bunch of dependencies (most notably the App Engine SDK) to the Python path.